api/v1_docs.inc

```diff
@@ -31,3 +31,10 @@ function api_v1_document(string $method, array $data, array $query_params): stri
     }
     return $faq_url;
 }
+
+function api_v1_dictionaries(string $method, array $data, array $query_params): array
+{
+    $dict_list = array_values(get_languages_with_dictionaries());
```

Comment:
I think if we returned the associated array directly from `get_languages_with_dictionaries()` instead of just the language this would be a more flexible endpoint.

Reply:
Done this and changed to asort because sort would re-assign keys. These keys could be useful to send back to simplify finding the aspell dictionaries although the code currently has to re-discover them from the language names.

Reply:
> Isn't this premature optimization? in the sense of sending extra information that we don't need but could possibly be useful in the future.

It is, but when designing APIs we sometimes need to do that, because after we publish them we can't change them in ways that break. So having a little extra data here is going to mean we don't have to create a v2 of the endpoint to add it later.